### ISSUE TITLE
How to check if loaded correctly?

### ISSUE THREAD

**Issue by HaleTom** [nvim-telescope/telescope-fzy-native.nvim#9]
I seem to be having exactly the same output both with and without loading this plugin.

I load it thus:

```
lua require('telescope').setup()
lua require('telescope').load_extension('fzy_native')
```

How can I check to ensure that the plugin is being used for both text and file sorting?

**Comment by fwolfst** [comment]
Agreed. A `:healtcheck fzy_native` would be great. I think my setup also does not work (loading the buildin-lua version does give me desired results, though).

**Comment by HaleTom** [comment]
@fwolfst last commit to this project was 3 years ago.

**Comment by fwolfst** [comment]
> [@fwolfst](https://github.com/fwolfst) last commit to this project was 3 years ago.

@HaleTom Never give up hope. ☮️ 

Btw this snippet loads the builtin lua version of the algorithm.
```lua
require("telescope").setup {
    -- [ ... ]
    pickers = {
          find_files = {
            sorter = require('telescope.sorters').get_fzy_sorter(), -- this is not the native implementation, is it?
          },
        }
}
```
